modules/installation-configuration-parameters.adoc

```diff
@@ -2689,30 +2680,33 @@ ifdef::agent[]
 |The name of the failure domain.
 |String
 
+|platform:
+  vsphere:
+    failureDomains:
+      region:
+|If you define multiple failure domains for your cluster, you must attach the tag to each vCenter datacenter. To define a region, use a tag from the `openshift-region` tag category. For a single vSphere datacenter environment, you do not need to attach a tag, but you must enter an alphanumeric value, such as `datacenter`, for the parameter.
+|String
+
 |platform:
   vsphere:
     failureDomains:
       server:
-|The fully qualified domain name (FQDN) of the vCenter server.
-|An FQDN such as example.com
-endif::agent[]
+|Specifies the fully-qualified hostname or IP address of the VMware vCenter server, so that a client can access failure domain resources. You must apply the `server` role to the vSphere vCenter server location.
+|String
 
 |platform:
   vsphere:
     failureDomains:
-      topology:
-        datastore:
-|Specifies the path to a vSphere datastore that stores virtual machines files for a failure domain. You must apply the `datastore` role to the vSphere vCenter datastore location.
+      template:
+|Specify the absolute path to a pre-existing {op-system-first} image template or virtual machine. The installation program can use the image template or virtual machine to quickly install {op-system} on vSphere hosts. Consider using this parameter as an alternative to uploading an {op-system} image on vSphere hosts. The parameter is available for use only on installer-provisioned infrastructure.
```

Comment:
Should we be consistent with Specifies/Specify?

Reply:
You used _this parameter in the previous sentence. Maybe use the same here for absolute clarity that you are referring to the same parameter? 

```suggestion
|Specify the absolute path to a pre-existing {op-system-first} image template or virtual machine. The installation program can use the image template or virtual machine to quickly install {op-system} on vSphere hosts. Consider using this parameter as an alternative to uploading an {op-system} image on vSphere hosts. This parameter is available for use only on installer-provisioned infrastructure.
```

Reply:
Updated.